### PR TITLE
Support shapefiles with empty attribute tables

### PR DESCRIPTION
**Pull request by mapmeld**
This PR tests compatibility with empty attribute tables returned by an updated parseDBF module https://github.com/calvinmetcalf/parseDBF/pull/8

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -26,7 +26,7 @@
     "lie": "^3.0.1",
     "lru-cache": "^2.7.0",
     "mocha": "^2.3.3",
-    "parsedbf": "0.1.1",
+    "parsedbf": "0.2.1",
```
Comment (calvinmetcalf) [inline package.json]: was publsihed as 0.1.2

**Comment by mapmeld** [comment]
@calvinmetcalf  fixed!

File: package.json [hunk 2]
```diff
@@ -1,6 +1,6 @@
 {
   "name": "shpjs",
-  "version": "3.3.1",
+  "version": "3.3.2",
```
Comment (calvinmetcalf) [inline package.json]: and don't touch this

**Comment by mapmeld** [comment]
@calvinmetcalf sure

**Comment by calvinmetcalf** [comment]
tests don't pass, will look into it latter today or tommorow

**Comment by mapmeld** [comment]
when I do ```mocha``` I have XMLHttpRequest issues, but ```npm test``` works. Let me know if you have a specific test failing

**Comment by calvinmetcalf** [comment]
the issue was there were weird duplicated dev deps vs prod deps, fixed and merged seems to work

**Pull request merged**